src/file-processor.ts

```diff
@@ -41,13 +40,21 @@ export async function* processDeploymentsInFile(
  */
 export async function* processDeploymentsInStream(
   stream: NodeJS.ReadableStream
-): AsyncIterable<DeploymentWithAuthChain> {
+): AsyncIterable<SyncDeployment> {
   for await (const line of processLineByLine(stream)) {
     const theLine = line.trim()
     if (theLine.startsWith('{') && theLine.endsWith('}')) {
-      const deployment = coerceEntityDeployment(JSON.parse(theLine))
-      if (deployment) {
-        yield deployment
+      const parsedLine = JSON.parse(theLine)
+      if (SnapshotSyncDeployment.validate(parsedLine)) {
+        yield parsedLine
+      } else if (PointerChangesSyncDeployment.validate(parsedLine)) {
```

Comment:
Are this the same entity type of the old snapshots?

Reply:
Yes. This line should be removed in the future when there are no old snapshots.